### PR TITLE
installer: test installed module run correctly in the system

### LINKED ISSUE #500: Installer: missing real test in different platforms
ref #497

currently, the installer has unit test only.

Did not test whether the installed module can run in the system.

- [ ] Windows
- [ ] MacOS
- [ ] Linux

I will work on this after #499 is merged

### PR DESCRIPTION
**Pull request by axetroy**
Make sure the installed modules are working correctly on the system.

closes #500 
wait for #510 be merged, and this PR should ready for review.

### CODE REVIEW

**Comment by bartlomieju** [comment]
@axetroy can you update the rest of installer test cases to install either `args.ts` or `echo.ts`? It makes no sense to install `file_server` during test pipeline - it downloads a lot of dependent files.

**Comment by axetroy** [comment]
> @axetroy can you update the rest of installer test cases to install either `args.ts` or `echo.ts`? It makes no sense to install `file_server` during test pipeline - it downloads a lot of dependent files.

@bartlomieju
`args.ts` and `echo.ts` used to test if `installer` passes parameters correctly to the installed module

Are you saying that the file_server module is no longer installed as a test?

**Comment by bartlomieju** [comment]
> Are you saying that the file_server module is no longer installed as a test?

It is, but it's a bad module to install in pipeline. There is no need to install a module which has a lot of deps.

File: installer/mod.ts [hunk 1]
```diff
@@ -241,6 +241,11 @@ export async function install(
     }
   }
 
+  // if install local module
+  if (!/^https?:\/\//.test(moduleUrl)) {
```
Comment (bartlomieju) [inline installer/mod.ts]: Yikes, this should have been merged in #499

Comment (axetroy) [reply installer/mod.ts]: I miss it. :)

Comment (axetroy) [reply installer/mod.ts]: I will add the relevant tests for the local module.

**Comment by axetroy** [comment]
move to #512

**Pull request closed without merging**